### DISCUSSION TITLE
Error Compiling FundMe.sol and PriceConverter.sol Contract

### DISCUSSION THREAD

**Discussion by Okiki-omisande** [Cyfrin/foundry-full-course-cu#2239] [Q&A] [answered]
Hi, I am trying to compile my `Fundme` contract and `PriceConverter` contract, but I keep getting this compilation error:

Source "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" not found: File import callback not supported

I have installed and compiled with both `chainlink-brownie-contract` version (1.1.1) and the latest version (1.2.0) but i keep getting the same error

Please, what could be causing this compilation issue?

Here are the screenshots of my `FundMe.sol`, `PriceConverter.sol` files, and my `lib` folder:

![Fundme.sol Screenshot](https://github.com/user-attachments/assets/69a9159b-0718-4649-82d7-5154c713495e) ![PriceConverter.sol Screenshot](https://github.com/user-attachments/assets/c4596a68-dc41-427e-8435-a560efe3b14e) ![Lib Folder Screenshot](https://github.com/user-attachments/assets/e7893fa4-4c98-425a-a845-a4eed5f84383)

**Comment by EngrPips** [comment, marked as answer]
Hello @Okiki-omisande, use `cltr + p` then type `AggregatorV3Interface` in the search bar that pops up and like that you will the accurate path for the `AggregatorV3Interface` file.

**Reply by Okiki-omisande** [reply]
Thank you so much, it fixed my issue

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
